### DISCUSSION TITLE
About outdated region files

### DISCUSSION THREAD

**Discussion by sonic3XE** [sonic3XE/evades-region-editor#3] [Announcements]
Ever since Kaluub posted a ZIP file that contain all regions in November 26, 2023, that is why I am going to make this issue to where some *region files* may be **outdated**. Do they usually post up-to-date region files (in ZIP) because last time posted was the said date. You are allowed to fork this repository for contribution of region files that can go up to date.

**Comment by Ino5559** [comment]
pls give me this zip file and if you can make panel for switch regions from evades.io pls

**Reply by Ino5559** [reply]
send zip file to gmail

**Reply by sonic3XE** [reply]
![image](https://github.com/sonic3XE/evades-region-editor/assets/137403814/413263b3-cecd-4cf1-9bf9-6234243017fb)

**Comment by sonic3XE** [comment]
related to #12 and may want to cache the region files from EvadesClassic
